analytics/analytics-ui/src/app/results/results.component.html

```diff
@@ -0,0 +1,17 @@
+<div style="margin-left:110px">
+
+  <!--Title-->
+  <h1>Results</h1>
+
+  <!--Back button-->
+  <button mat-raised-button color="secondary" id="execute-analytic" (click)="goback()">BACK
```

Comment:
This should be a link, not a button. I don't think you really need a newline in the button